src/model/update.type.model.ts

```diff
@@ -3,6 +3,6 @@ export const UpdateKey = "update";
 export const UpdateKeys: string[] = [ "date_of_creation" ];
 
 export interface Update {
-    date_of_creation?: string
+    date_of_creation?: string,
+    fetchedBoMoData?: true
```

Comment:
does it matter that keys are mixture of snake and camel case?

Reply:
Not in this case as just for session but will align it.